### DISCUSSION TITLE
uso de bibliotecas

### DISCUSSION THREAD

**Discussion by rominaarancibia** [IIC2133-PUC/2024-1#81] [Tarea 1] [answered]
Hola! Esperando estén bien. Quería preguntar si el uso de bibliotecas está permitido.

**Comment by rominaarancibia** [comment]
como priority_queue

**Comment by benjavicente** [comment, marked as answer]
Hola! No pueden usar librerías para implementar las EDDs principales de la tarea.

**Reply by MaxAl100** [reply]
Pero sí para hacer cálculos? como por ejemplo potencias (10 elevado a 5 o cosas así)
(con math.h)

**Reply by benjavicente** [reply]
Eso si.
